cluster/gce/windows/k8s-node-setup.psm1

```diff
@@ -1732,6 +1732,7 @@ $FLUENTD_CONFIG = @'
       format json
       time_key time
       time_format %Y-%m-%dT%H:%M:%S.%NZ
+      keep_time_key
```

Comment:
would it be possible to check if this is needed for the fluentbit section as well.

Reply:
+1

Reply:
Synced offline with Barni. We found that the equavelent Time_Keep On setting is already added to fluentbit config.